packages/components/ng-ovh-sso-auth/src/provider.js

```diff
@@ -1,3 +1,5 @@
+import 'whatwg-fetch';
```

Comment:
Why not consider using it as a PeerDependency instead? (same from ng-ovh-sso-auth-modal-plugin component).

Because this dependency is always imported from the host application.

```sh
$ git grep "whatwg-fetch" -- "*.js"
packages/manager/apps/dedicated/client/app/index.js:import 'whatwg-fetch';
packages/manager/apps/hub/src/index.js:import 'whatwg-fetch';
packages/manager/apps/public-cloud/src/index.js:import 'whatwg-fetch';
packages/manager/apps/telecom/src/app/index.js:import 'whatwg-fetch';
packages/manager/apps/web/client/app/index.js:import 'whatwg-fetch';
```

Missing `packages/manager/apps/cloud/..`

Reply:
The `cloud` manager don't use the `bootstrapApplication` from https://github.com/ovh/manager/blob/master/packages/manager/modules/core/src/index.js#L261, so no `fetch` used.

`whatwg-fetch`is a `fetch` polyfill so I added it in applications (not required, it depends on the application)